metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs and configures Gluster servers and clients'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '5.1.0'
+version          '5.1.1'
```

Comment:
why? I think that's a "feature" fix, so it's easier to bump version to point which version works :)